packages/app/src/components/farm/farm.tsx

```diff
@@ -126,7 +127,7 @@ const Farm: FC<Props> = ({ protocolName, symbol, tokenBalance }) => {
                 <ApproveToken
                   symbol={symbol}
                   spender={addresses[protocolName].tenderizer}
-                  token={contracts[protocolName].token}
+                  token={contracts[protocolName].lpToken as unknown as ERC20}
```

Comment:
'as unknown' is probably needed, but just for my knowledge, why?

Reply:
`Conversion of type 'LiquidityPoolToken' to type 'ERC20' may be a mistake because neither type sufficiently overlaps with the other.` - not really sure what we have on ERC20 we don't on LPToken, but it shouldn't matter here

Reply:
Oh, can't we use it as 'LiquidityPoolToken' itself which should have all the functions we need already right?

Reply:
LiquidityPoolToken itself is an extension of ERC20 so the reverse dependency is why it doesn't overlap.